bin/identify_redundant_fams.py

```diff
@@ -26,14 +26,27 @@ def parse_args(args=None):
         type=str,
         help="TSV hmmsearch domtbl out results for filtering.",
     )
+    parser.add_argument(
+        "--skip_family_redundancy_removal",
+        action="store_true",
+        help="If set, skip filtering of similar families above redundancy threshold.",
+    )
     parser.add_argument(
         "-l",
-        "--length_threshold",
+        "--redundancy_length_threshold",
         required=True,
```

Comment:
Confused about why this is required if you can now skip family redundancy removal (see argument right before it)... maybe they're not the same and I'm misunderstanding though